internal/cmd/broker/delete_broker_test.go

```diff
@@ -44,9 +46,20 @@ var _ = Describe("List brokers command test", func() {
 		return commandToRun.Execute()
 	}
 
+	Context("when existing broker is being deleted forcefully", func() {
+		It("should list success message", func() {
```

Comment:
Can we also have a negative test for this - when the -f parameter is not provided?